bin/cliain/src/main.rs

```diff
@@ -93,6 +93,13 @@ enum Command {
         to_account: String,
     },
 
+    /// Send new runtime (requires sudo account)
+    UpdateRuntime {
```

Comment:
There's a convention in `cliain` now that command line argument resembles 1:1 (apart `ChangeMembers`) the substrate chain call, shall we heep it here and call this argument `SetCode`?

Reply:
Okay.